Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -74,8 +74,10 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
             textValueOfCellType[type.index] = hasValue ? value : (!isMandatory ? "" : nil)
             errorFlagOfCellType[type.index] = hasValue ? false : isMandatory
         }
-        if let country = country, country.iso3166Alpha2.isEmpty {
-            self.country = Country(iso3166Alpha2: Locale.current.regionCode ?? "", dialingCode: nil)
+        if country == nil,
+           let regionCode = Locale.current.regionCode,
+           let deviceCountry = Country.from(iso3166Alpha2: regionCode) {
```

Comment:
Just an idea, but we could replace the static method `from` with an initialiser, making the syntax slightly simpler to
```swift
Country(from: regionCode)

/// if we drop initialiser from dialingCode, this would be safe
init(from iso3166Alpha2: String)

/// if we keep dialingCode initialiser we want to avoid ambiguity
init(iso3166Alpha2: String)
```